L1Trigger/Phase2L1ParticleFlow/python/l1ctLayer1_cff.py

```diff
@@ -28,6 +28,8 @@
     regionizerAlgo = cms.string("Ideal"),
     pfAlgo = cms.string("PFAlgo3"),
     puAlgo = cms.string("LinearizedPuppi"),
+    vtxRes      = cms.double(0.333),

```

Comment:
vtxRes no longer needed or used anywhere, so remove from here and the rest of the file